geopy/distance.py

```diff
@@ -284,6 +316,8 @@ def destination(self, point, bearing, distance=None): # pylint: disable=W0621
 
         return Point(units.degrees(radians=lat2), units.degrees(radians=lng2))
 
+GreatCircleDistance = great_circle
```

Comment:
This line seems to be redundant: there's the same one at the bottom of this module.